modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheIdMessage.java

```diff
@@ -52,6 +54,7 @@ public void cacheId(int cacheId) {
 
     /** {@inheritDoc} */
     @Override public boolean writeTo(ByteBuffer buf, MessageWriter writer) {
+        // TODO: Safe to remove only after all inheritors have migrated to the new ser/der scheme (IGNITE-25490).
```

Comment:
May be we should move it to the method level or make comment more clear? Now it points to next line.